modules/integration/src/test/java/org/apache/fluo/integration/impl/OracleIT.java

```diff
@@ -226,7 +226,7 @@ public void singleOracle_goesAwayAndComesBack() throws Exception {
     sleepWhileConnected(oserver);
 
     int count = 0;
-    while (count < 5 && client.getOracle() != null) {
+    while (count < 20 && client.getOracle() != null) {
```

Comment:
I changed to a timeout rule.  Other test in the class wait for conditions and will benefit from this change.  I am going to open an issue to use timeout rule in all ITs.